client/trino-client/src/main/java/io/trino/client/ResultRows.java

```diff
@@ -56,6 +62,12 @@ public Iterator<List<Object>> iterator()
             {
                 return values.iterator();
             }
+
+            @Override
+            public String toString()
+            {
+                return "ResultRows{values=" + values + "}";
```

Comment:
not sure if that it useful. Do you really want to print out all values?

Reply:
It's only used in tests anyway